doc/howto/cluster/cmd_argument_en.md

```diff
@@ -138,3 +141,20 @@ Your workspace may looks like:
 
 - `train_data_dir`: containing training data. Mount from storage service or copy trainning data to here.
 - `test_data_dir`: containing testing data.
+
+## Async SGD Update
+
+You can modify the parameters of `optimize` to support Async SGD update, for example:
```

Comment:
I don't see a parameter named `optimize` in the following code snippet. Do you mean

> We can set some parameters of the optimizer to make it support async SGD update.  For example, we can set the `is_async` and `async_lagged_grad_discard_ratio` of the `AdaGrad` optimizer:

Reply:
Thanks, this description is clearer.